### DISCUSSION TITLE
Discord OAUTH Returning Two Codes

### DISCUSSION THREAD

**Discussion by Moving-Light** [requarks/wiki#3289] [Error / Bug Report]
**Describe the bug**
When going through the Discord Authenticator, the return URI has two `?code=` definitions, causing it to not recognize the code.

**To Reproduce**
1. Set up the discord authenticator following the [Discord Authenticator Docs](https://docs.requarks.io/auth)
2. Go to log in with Discord like normal
3. See error

**Expected behavior**
Only one code will be returned, meaning a successful log in (tested by deleting one of the code segments)

**Screenshots**
![image](https://user-images.githubusercontent.com/28193957/103142356-d6273380-46cf-11eb-9526-a116f63cac10.png)
![image](https://user-images.githubusercontent.com/28193957/103142357-dcb5ab00-46cf-11eb-876e-cfa91887df58.png)

**Host Info (please complete the following information):**
 - OS: Ubuntu 18.04, Docker
 - Wiki.js version: 2.5.170
 - Database engine: postgres 11.10

**Comment by ghost** [comment]
I could not replicate the error, check if it still occurs, because its works for me.

**Comment by Moving-Light** [comment]
It is still happening.  Just tried on two different browsers.
